src/main/java/org/codehaus/mojo/taglist/TagListReport.java

```diff
@@ -520,17 +528,39 @@ String getExcludesCommaSeparated() {
         }
     }
 
+    void setSourceFileLocale(String sourceFileLocale) {
+        this.sourceFileLocale = sourceFileLocale;
+    }
+
     /**
      * Returns the Locale of the source files.
      *
      * @return The Locale of the source files.
      */
-    public Locale getLocale() {
-        // The locale string should never be null.
-        if (sourceFileLocale == null) {
-            sourceFileLocale = DEFAULT_LOCALE;
+    public Locale getSourceFileLocale() {
+        String[] items = sourceFileLocale.split("_");
+        if (items.length > 3) {
+            getLog().warn("Invalid java.util.Locale format '" + sourceFileLocale
+                    + "' for 'sourceFileLocale' use ROOT locale");
```

Comment:
Why not the default locale (value) since your default value is `"en"`, not `ROOT`.